flash_examples/finetuning/image_detection.py

```diff
@@ -0,0 +1,26 @@
+import flash
+from flash.vision.detection.data import ImageDetectionData
+from flash.vision.detection.dataset import coco128_data_download
+from flash.vision.detection.model import ImageDetector
+
+# 1. Download the data
+coco128_data_download("data/")
+
+# 2. Load the Data
+datamodule = ImageDetectionData.from_coco(
```

Comment:
Can we add a filter to coco ? I know it is pretty long to load :)

Reply:
It's a filtered coco dataset. It has only 128 images.

Reply:
@tchaton